configs/components/runtime-pe-installer.rb

```diff
@@ -19,7 +19,11 @@
 
   else # Linux and Solaris systems
     libbase = platform.architecture =~ /64/ ? 'lib64' : 'lib'
-    libdir = "/opt/pl-build-tools/#{libbase}"
+    if platform.name =~ /el-8/
+      libdir = "/usr/#{libbase}"
+    else
+      libdir = "/opt/pl-build-tools/#{libbase}"
```

Comment:
I don't think we need to continue using `pl-build-tools` for any platform that we build.

Reply:
@McdonaldSeanp @caseywilliams Should we remove all use of `pl-build-tools` from the runtimes?

Reply:
pl-build-tools are still used here https://github.com/puppetlabs/puppet-runtime/blob/8736f3c83e8e39a683cacfe8195f9e5b9d87c12d/configs/components/runtime-pdk.rb#L35-L43

Reply:
Alright... I guess we'll keep using it? Hopefully it doesn't end up breaking suddenly later.